cmd/juju/application/deployer/bundlehandler.go

```diff
@@ -48,6 +48,13 @@ import (
 	"github.com/juju/juju/storage"
 )
 
+// localCharmChannel is used to identify a standard channel that all local
+// charms will adhere to. THis isn't actually required, but it makes things a
+// lot simpler when using a selecting algorithm based on channels for charms.
+var localCharmChannel = corecharm.Channel{
+	Risk: corecharm.Risk("stable"),
```

Comment:
Could this (the stable risk value) perhaps be something exported by `corecharm`?